amenity-points.mss

```diff
@@ -40,12 +40,12 @@
   [feature = 'tourism_alpine_hut'][zoom >= 13],
   [feature = 'tourism_wilderness_hut'][zoom >= 13],
   [feature = 'amenity_shelter'][zoom >= 16] {
-    marker-file: url('symbols/shelter.svg');
```

Comment:
Are these file moves from you or a merge / git artefact?
This should be done in a seperate PR.